src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/integration/DraftAdjudicationIntTest.kt

```diff
@@ -123,6 +123,29 @@ class DraftAdjudicationIntTest : IntegrationTestBase() {
       .jsonPath("$.draftAdjudication.incidentStatement.modifiedByDateTime").exists()
   }
 
+  @Test
```

Comment:
Could do with a negative test - when the save fails in Prison API.

But that's quite a lot of work, so not now.

Reply:
Agreed. I'll write one now and submit in the next PR